bootstrap.sh

```diff
@@ -32,6 +32,7 @@ install-requirements() {
 
   case "$DOKKU_DISTRO" in
     debian|ubuntu)
+      add-apt-repository universe > /dev/null
```

Comment:
What happens if universe is already enabled? I also wonder we should be sending output to `/dev/null`.

Reply:
I've tested this locally and doesnt seem to effect my installation if already enabled... (and the > /dev/null stuff is just there to follow suit as the `apt-get update -qq > /dev/null` also has this. Since this is a minor addition, it wouldn't hurt... (i thought)